R/plate-utils.R

```diff
@@ -77,12 +77,32 @@ init_meta <- function(plate) {
   if (is.null(plate_meta)) {
     plate_meta <- DEFAULT_PLATE_META
   } else {
-    meta_cols_keep <- c("well", "sample")
+    meta_cols_keep <- c("well", "sample", "target_ch1", "target_ch2")
     tryCatch({
+      
+      if (colnames(plate_meta)[5] == 'typeassay' | 
+          colnames(plate_meta)[5] == 'targettype'){
+        # different versions of QuantaSoft
+        if (colnames(plate_meta)[5] == 'typeassay'){
+          plate_meta %<>% dplyr::mutate(target = assay, 
```

Comment:
Could you try to use the NSE version `mutate_` (same for `group_by_` and `summarise_` below)